packages/desktop-client/src/components/reports/ReportOptions.tsx

```diff
@@ -21,10 +21,11 @@ const groupByOptions = [
 ];
 
 const dateRangeOptions = [
-  { description: '1 month', name: 1 },
-  { description: '3 months', name: 2 },
-  { description: '6 months', name: 5 },
-  { description: '1 year', name: 11 },
+  { description: 'This month', name: 0 },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this actually correct?

If I open the page on 2nd of January: will it show only 2 days worth of data (1st - 2nd Jan)? Or would it show the last ~30 days?

Reply:
It's by month where monthDiff = 0. So it would show only 2 days. I plan to add an interval selector where you can view data by day or week or year etc. At that point, I'll also add new range options (based on which interval you select) like "last 30 days", "last 7 days", "this week", "last week", etc.